fdm-core/src/cultivation.ts

```
@@ -674,7 +689,7 @@
             )
 
         const cultivationPlan = cultivations.reduce(
-            (acc: cultivationPlanType[], curr: any) => {
+            (acc: CultivationPlan[], curr: any) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)